src/main/java/com/rtb/projectmanagementtool/user/UserController.java

```diff
@@ -12,8 +12,8 @@ public UserController(HashSet<UserData> users) {
   }
 
   // get users+data to direct/display user's profiles later
-  public HashSet<UserData> getEveryUser() {
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  public HashSet<UserData> getEveryUser(DatastoreService datastore) {
+    // DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     HashSet<UserData> users = new HashSet<>();
```

Comment:
Watch out for variable scoping.  This `users` is a local variable, and not the class member (not `this.users`).  This is not necessarily bad though.  Maybe if you don't have a good use case for holding on to a class member, consider removing.